dll/win32/syssetup/syssetup.spec

```
@@ -62,7 +62,7 @@
 @ stub SetupOobeInitPostServices
 @ stub SetupOobeInitPreServices
 @ stub SetupPidGen3
-@ stub SetupQueryRegisteredOsComponent
+@ stdcall -stub SetupQueryRegisteredOsComponent()
```

```suggestion
@ stdcall -stub SetupQueryRegisteredOsComponent(ptr ptr ptr)
```
This might not be entirely correct, but at least the amount of arguments should be fine :)

---

Committed. Thanks @learn-more.
If this is more correct, I'm OK with this change. You know better. :)